tests/environment.py

```diff
@@ -106,9 +109,24 @@ def create():
             Station.__mac_base = 0
         return Station(mac)
 
+    def connect(self, bssid):
+        '''Simulate station connection'''
+        self.bssid = bssid
+
+    def disconnect(self):
+        '''Simulate station disconnection'''
+        self.bssid = None
+
+    def connected(self, bssid=None) -> bool:
```

Comment:
I was going to comment on how I don't like this connect and disconnect methods (they should be private), but then I wasn't able to find how it is used to begin with...

Reply:
Right, I meant to use them and eventually didn't, removed.